service/patch.go

```diff
@@ -103,11 +104,17 @@ func (uis *UIServer) schedulePatch(w http.ResponseWriter, r *http.Request) {
 	}
 
 	patchUpdateReq := patchVariantsTasksRequest{}
-
 	if err = util.ReadJSONInto(util.NewRequestReader(r), &patchUpdateReq); err != nil {
 		http.Error(w, err.Error(), http.StatusBadRequest)
 		return
 	}
+	grip.InfoWhen(len(patchUpdateReq.Tasks) > 0 || len(patchUpdateReq.Variants) > 0, message.Fields{
```

Comment:
Is this log line still needed?

Reply:
This log line is to assist me in the future refactor. I can't find anything using these fields, but I want to be sure.